.github/workflows/release.yml

```
@@ -410,6 +410,13 @@ jobs:
         # come from the static `app/src-tauri/tauri.conf.json`.
         uses: actions/github-script@v7
         env:
+<<<<<<< Updated upstream
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)